src/components/Header.js

```diff
@@ -651,14 +651,17 @@ class Header extends Component {
               <Icon className="material-icons">perm_media</Icon>
             </IconButton>
           </Tooltip>
-          <MyrTour />
+          <MyrTour 
+            isEditorClosed={this.props.scene.settings.viewOnly}
```

Comment:
For consistency sake, while you're making the other edit can isEditorClosed be renamed to viewOnly? That way it's one term in all places.